### DISCUSSION TITLE
Export Button?

### DISCUSSION THREAD

**Discussion by ttate1978** [sqlitebrowser/sqlitebrowser#3114] [General]
Hello,

Its quite possible that I may be looking over the button but I can't find the export button on SQLite. Is there an export function?

**Comment by chrisjlocke** [comment]
To export what to what?  Export a table?  Export a query?  Export settings?

![image](https://user-images.githubusercontent.com/7872218/179292138-d7587ebb-c083-460a-a5c1-6a9447bc61b7.png)

If you want to export the whole database to a SQL file, then click 'File', 'Export' and select 'Database to SQL file'.  You'll also be able to export a table to CSV or JSON.

![image](https://user-images.githubusercontent.com/7872218/179292347-22d0cae7-ffcd-41b2-8ba9-a2a7a3f52622.png)

To export a query in a tab, click the 'Save SQL file' button.  

![image](https://user-images.githubusercontent.com/7872218/179292534-3baf3efa-dfcc-41ea-b804-eefe3b972207.png)

You can also export the results to a view, CSV or JSON.

![image](https://user-images.githubusercontent.com/7872218/179292608-ac33fadf-07da-46f1-888a-e7c8ee753eeb.png)

If you click 'Edit' and 'Preferences' you can export settings.
